src/universal/containers/EditingStatus/EditingStatusContainer.js

```diff
@@ -79,22 +79,33 @@ export default class EditingStatusContainer extends Component {
     className: PropTypes.object,
     editors: PropTypes.any,
     outcomeId: PropTypes.string,
-    updatedAt: PropTypes.instanceOf(Date)
+    updatedAt: PropTypes.instanceOf(Date),
+    createdAt: PropTypes.instanceOf(Date)
   };
 
   constructor(props) {
     super(props);
-    const {active, editors, updatedAt} = this.props;
+    const {active, editors, updatedAt, createdAt} = this.props;
+    const timestampType = 'updatedAt'; // Should pull this default from storage instead
+    const timestamp = timestampType === 'updatedAt' ? updatedAt : createdAt;
     this.state = {
-      editingStatus: makeEditingStatus(editors, active, updatedAt)
+      timestampType,
+      editingStatus: makeEditingStatus(editors, active, timestamp, timestampType)
     };
+    this.toggleTimestamp = this.toggleTimestamp.bind(this);
```

Comment:
You could even declare the function as : 
```
toggleTimestamp = () => {
...
}
```
instead of 
```
toggleTimestamp() {
...
}
```

So you don't have to encapsulate the function in a arrow function everytime you pass it to a component...

Cheers 🎉